src/components/DeploymentsModal.tsx

```diff
@@ -130,6 +125,16 @@ export const DeploymentsModal = ({ handleCloseModal, isModalOpen }: IDeployments
     columnIndex,
   });
 
+  function search(items: any[]) {
```

Comment:
Probably we can assume that the search items will just be an array of strings, right? so maybe for the type let's put `items: string[]`, unless it gives you issues.

Reply:
this is an array of IDeployment's, so to make the function more specific changed to this type